ikarus/controlroutines/pathfollowingfunctions.hh

```diff
@@ -74,13 +74,11 @@ struct ArcLength
    * \param args The subsidiary function arguments.
    */
   void operator()(SubsidiaryArgs& args) const {
-    if (psi) {
-      const auto root = sqrt(args.DD.squaredNorm() + psi.value() * psi.value() * args.Dlambda * args.Dlambda);
+    if (psi != std::numeric_limits<double>::infinity()) {
```

Comment:
I removed `psi` as an optional variable. This is because `initialPrediction` is not called for the dummy execution of the code to save the state of the initial configuration, which then throws an error when the `else` branch is called. But, still to ensure a reasonable update of `psi`, I defaulted its value to infinity, such that numerical problems arise if the it is not updated later at the first load step.

Reply:
I changed it back to the original version as now the interface of control routines doesn't solve the system of equations when executing the dummy steps to save the information of the initial configuration.